api/services/tasks/v1/tasks.proto

```diff
@@ -196,6 +196,8 @@ message CheckpointTaskRequest {
 	string container_id = 1;
 	string parent_checkpoint = 2;
 	google.protobuf.Any options = 3;
+	bool export_to_archive = 4;
+	string location = 5;
```

Comment:
The purpose of `location` is not very obvious.
Would it be possible to use `archive_location` or something similar instead?

Reply:
I was using the same names as in the CRI API, but I can rename this to whatever is best for here.